scripts/test_zeth_zecale

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+# TODO: Remove this once the issue of the number of nested inputs is resolved.
+echo "============================================================="
+echo "                         !!WARNING!!"
+echo "============================================================="
+echo "This test requires a zecale aggregator configured for 9 nested"
+echo "inputs. For now, this must be changed in the aggregator source:"
+echo "  aggregator_server/aggregator_server.cpp"
+echo ""
+
+set -x
+set -e
+
+ROOT_DIR=`pwd`
+
+# 1 - user_name
+# 2- (optional) network name
+function setup_user() {
+    mkdir -p $1
+    pushd $1
+    [ -e eth-network ] || zeth_helper eth-gen-network-config $2
+    [ -e eth-address ] || \
+        (zeth_helper eth-gen-address && zeth_helper eth-fund)
+    [ -e zeth-address.priv ] || zeth gen-address
+    popd
+}
+
+mkdir -p _test_zeth_zecale_data
+pushd _test_zeth_zecale_data
+
+# Create user zecale deployer
+
+setup_user zecale_deployer
+
+# Deploy zecale
+
+pushd zecale_deployer
+    [ -e aggregator_vk.json ] || zecale get-verification-key
```

Comment:
As far as I can see, the same sub-command (i.e. `get-verification-key`) has a different behavior whether it is used with the `zeth` cli or with the `zecale` cli as shown by the logs below:
```console
(env) 18:18:45 ~/D/z/client ❯❯❯ zeth get-verification-key --help
Usage: zeth get-verification-key [OPTIONS]

  Command help text.

Options:
  -o, --vk-out TEXT  Output file
  --help             Show this message and exit.
(env) 18:18:48 ~/D/z/client ❯❯❯ zecale get-verification-key --help
Usage: zecale get-verification-key [OPTIONS]

  Get the aggregator (wrapping) verification key from the aggregation server
  and write to a file.

Options:
  -f, --verification-key-file TEXT
                                  Write verification key to this file
                                  (default: aggregator-vk.json)
  --help                          Show this message and exit.
(env) 18:18:49 ~/D/z/client ❯❯❯

```

I think that would make sense to align the behaviors here by using consistent flags (long and short) and potentially providing a default value for the zeth command as well.

Reply:
renamed the zecale flags

Reply:
Thanks